src/App.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import Listings from './sections/Listings';
 import 'antd/dist/reset.css';
-import './styles/index.css';
+import './styles/index.scss';
```

Comment:
Can you [BEM](https://css-tricks.com/bem-101/) in a scss file to apply styles

Reply:
https://github.com/igorizviekov/mt-booking_app/commit/a8c28341f2fd96cd75a42ba8a1e8e714e4184007

Reply:
update scss file with refactored nested class names to be` &__{{ className }}`